src/main/resources/assets/chatonlostfocus/lang/zh_tw.json

```diff
@@ -0,0 +1,11 @@
+{
+  "keybind.chatonlostfocus.onOffBind": "開啟／關閉模組",
+  "category.chatonlostfocus.chatonlostfocus": "Chat on Lost Focus 失去焦點時發送訊息",
+  "overlay.chatonlostfocus.on": "§a[失焦時發送訊息] 模組已啟用",
+  "overlay.chatonlostfocus.off": "§c[失焦時發送訊息] 模組已停用",
```

Comment:
Isn't it better to make mod name in braces in english? like searching for mod for example in binds or mod menu would be nicer i guess

Reply:
Sure i will change now